src/main.rs

```diff
@@ -185,3 +196,53 @@ fn parse_args() -> Result<LndCfg, ArgsError> {
 
     Ok(LndCfg::new(address, cert_file, macaroon_file))
 }
+
+/// Sets the onion messaging feature bit (described in this PR:
+/// https://github.com/lightning/bolts/pull/759/), to signal that we support
+/// onion messaging. This needs to be done every time we start up, because LND
+/// does not currently persist the custom feature bits that are set via the RPC.
+async fn set_feature_bit(mut client: Client) {
+    let onion_feature_bit = 39;
+    let update = tonic_lnd::peersrpc::UpdateFeatureAction {
+        action: i32::from(tonic_lnd::peersrpc::UpdateAction::Add),
+        feature_bit: onion_feature_bit,
+    };
+    let feature_updates = vec![update];
+    let address_updates = vec![];
+
+    let resp = client
+        .peers()
+        .update_node_announcement(tonic_lnd::peersrpc::NodeAnnouncementUpdateRequest {
+            feature_updates,
+            color: String::from(""),
+            alias: String::from(""),
+            address_updates,
```

Comment:
Can use `.Default::default()` do that we don't need to fill in empty values here.

Reply:
oh that's super nifty